.github/workflows/preview-release-on-comment.yml

```
@@ -0,0 +1,99 @@
+name: Preview Release on Comment
+
+permissions:
+  contents: read
+  pull-requests: write
+  issues: write
+  statuses: write
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  release:
+    if: github.event.issue.pull_request && startsWith(github.event.comment.body, '[preview_deployment]')
+    runs-on: ubuntu-latest
+
+    timeout-minutes: 15
+
+    steps:
+      # Get GitHub token via the CT Changesets App
+      - name: Generate GitHub token (via CT Changesets App)
+        id: generate_github_token
+        uses: tibdex/github-app-token@v2.1.0
+        with:
+          app_id: ${{ secrets.CT_CHANGESETS_APP_ID }}
+          private_key: ${{ secrets.CT_CHANGESETS_APP_PEM }}
+      
+      - name: Get branch of PR
+        uses: xt0rted/pull-request-comment-branch@v1
+        id: comment-branch
+      
+      - name: Ensure branch name matches 'preview/**'
+        run: |
+          BRANCH_NAME=${{ steps.comment-branch.outputs.head_ref }}
+          if [[ ! "$BRANCH_NAME" =~ ^preview/ ]]; then
+            echo "Branch name does not match 'preview/**'. Exiting workflow."
+            exit 1
+          fi
+        
+      - name: Set latest commit status as pending
+        uses: myrotvorets/set-commit-status-action@master
+        with:
+          sha: ${{ steps.comment-branch.outputs.head_sha }}
+          token: ${{ steps.generate_github_token.outputs.token }}
+          status: pending
+
+      - name: Checkout PR branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ steps.comment-branch.outputs.head_ref }}
+          token: ${{ steps.generate_github_token.outputs.token }}   
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          registry-url: https://registry.npmjs.org/
+          scope: "@commercetools"
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Installing dependencies and building packages
+        uses: ./.github/actions/ci
+
+      - name: Publishing preview releases to npm registry
+        run: |
+          BRANCH_NAME=${{ steps.comment-branch.outputs.head_ref }}
+          PREVIEW_TAG=$(echo "$BRANCH_NAME" | sed -e 's/^preview\///')
+          git checkout ${{ steps.comment-branch.outputs.head_ref }}
```

Not that it matters much but I think since we already declared BRANCH_NAME, maybe we can use it in the checkout line. 
```suggestion
          git checkout ${BRANCH_NAME}
```

---

This line was removed.